com.realm667.WolfenDoom_Blade_of_Agony.json

```diff
@@ -0,0 +1,126 @@
+{
+	"app-id": "com.realm667.WolfenDoom_Blade_of_Agony",
+	"runtime": "org.freedesktop.Platform",
+	"sdk": "org.freedesktop.Sdk",
+	"runtime-version": "1.6",
+	"command": "gzdoom.sh",
+	"rename-icon": "m_doom",
+	"finish-args": [
+		"--device=dri",
+		"--socket=wayland",
+		"--socket=x11",
+		"--share=ipc",
+		"--socket=pulseaudio",
+		"--env=DOOMWADDIR=/app/share/games/doom",
+		"--filesystem=xdg-config/gzdoom"
```

Comment:
This shouldn't be here, use `--persist` when XDG isn't respected.

Reply:
Done